.gitignore

```diff
@@ -9,11 +9,9 @@
 /Distribution/Backups
 /Distribution/Saves
 /Distribution/docs
-/Distribution/zlib.dll
 /Distribution/libz.dylib
 /Distribution/libz.so
-/Distribution/ZLib.Bindings.dll
-/Distribution/Microsoft.Toolkit.HighPerformance.dll
+/Distribution/*.dll
```

Comment:
Hmm I am debating if we should do this. My biggest question is how will people consume this repo. If they fork it and start dropping files in, I don't want them to be excluded.

Reply:
That's a fair point but still, dll files in the build output directory should never be commited into version control, don't you think? Moreover, there's already the `.csproj` files which is a manifest of what libraries and what specific versions are needed, and that one is already checked in.

Reply:
I guess this is fine. Let's do the same for .so and .dylib files.

Reply:
Done!